test/integration/01-setup/01-lightning-channels.spec.ts

```diff
@@ -0,0 +1,182 @@
+import { once } from "events"
+import { find } from "lodash"
+import { MainBook } from "src/mongodb"
+import { lndFeePath } from "src/ledger/ledger"
+import {
+  checkIsBalanced,
+  closeChannel,
+  getChannels,
+  getWalletInfo,
+  lnd1,
+  lnd2,
+  lndOutside1,
+  lndOutside2,
+  mineBlockAndSync,
+  openChannelTesting,
+  setChannelFees,
+  subscribeToChannels,
+  subscribeToGraph,
+  updateRoutingFees,
+  waitFor,
+  waitUntilSync,
+} from "test/helpers"
+import { offchainLnds, onChannelUpdated, updateEscrows } from "src/lndUtils"
+import { sleep } from "src/utils"
+
+jest.mock("src/realtimePrice", () => require("test/mocks/realtimePrice"))
+
+//this is the fixed opening and closing channel fee on devnet
+const channelFee = 7637
+const lnds = [lnd1, lnd2, lndOutside1, lndOutside1]
+let channelLengthMain, channelLengthOutside1
+
+beforeEach(async () => {
+  await waitUntilSync({ lnds })
+  channelLengthMain = (await getChannels({ lnd: lnd1 })).channels.length
+  channelLengthOutside1 = (await getChannels({ lnd: lndOutside1 })).channels.length
+})
+
+afterEach(async () => {
+  await checkIsBalanced()
+})
+
+// Setup the next network
+// lnd2 <- lnd1 <-> lndOutside1
```

Comment:
is it intented the fact we have two arrow on this side: `lnd1 <-> lndOutside1`?
I would expect one node to open the channel. maybe it's because there are 2 channels (just trying to guess from the comment, not looking at the code yet)

Reply:
I just added in that way because the setup open two channels.

[lnd -> lndOutside1](https://github.com/dolcalmi/galoy/blob/refactor_test_stg3a/test/integration/01-setup/01-lightning-channels.spec.ts#L63)
[lndOutside1 -> lnd](https://github.com/dolcalmi/galoy/blob/refactor_test_stg3a/test/integration/01-setup/01-lightning-channels.spec.ts#L89)